psamm/command.py

```diff
@@ -91,6 +92,18 @@ def init_parser(cls, parser):
     def run(self):
         """Execute command"""
 
+    def argument_error(self, msg):
+        """Raise error indicating error parsing an argument."""
+        raise CommandError(msg)
+
+    def fail(self, msg, exc=None):
+        """Exit command as a result of a failure."""
+        if exc is not None:
+            logger.debug(msg, exc_info=exc)
+        else:
+            logger.debug(msg)
+        sys.exit(1)
+
```

Comment:
I think you misunderstood this... the `msg` should be logged at the `error` level otherwise a normal user will not know what happened. In other words, `logger.error` should be used to log `msg`. In addition, if `exc` is provided, the exception should be logged with `logger.debug`. This is to provide additional log messages about the exception traceback so that a developer can trace the source of the failure.